pages/index.tsx

```diff
@@ -21,7 +21,8 @@ type PageProps = {
 const Page: NextPage<PageProps> = ({ projects, partners }) => {
   const theme = useContext(ThemeContext);
   return (
-    <Layout>
+    // TODO: Dynamic display of banner based on URL parameter
+    <Layout showBanner={true}>
```

Comment:
Jak to chcem nejlíp udělat? Může ten banner být pouze na titulce? Ten parametr z URL myslím můžem přečíst přes router hook, viz [next/router](https://nextjs.org/docs/api-reference/next/router). Něco jako:

```typescript
import { useRouter } from 'next/router'

const router = useRouter()
const displayBanner = !!router.query.magicBannerSwitch;
```

Reply:
Dobry postreh, ten banner bych dal do <Layout> komponenty, at muze byt vsude. My necilime specificky na homepage (to je mozna i pravidlo, na homepage obecne necilime), spis na obsah projekty, portal, blog.

Reply:
Hotovo v a222070.